### PR TITLE
test: isolate uibuilder test

### PR DESCRIPTION
**Pull request by sobolk**
#### Description of changes

uibuilder test calls `pkill -f react` which might affect other tests running in parallel if they start react apps.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jhockett — APPROVED** [review]
Nice!

File: codebuild_specs/e2e_workflow_generated.yml [hunk 1]
```diff
@@ -825,7 +826,16 @@ batch:
...
           TEST_SUITE: src/__tests__/auth_2e.test.ts
-          CLI_REGION: us-east-1
+          CLI_REGION: ap-southeast-1
+      depend-on:
+        - upb
+    - identifier: l_uibuilder
+      buildspec: codebuild_specs/run_e2e_tests_linux.yml
+      env:
+        variables:
+          compute-type: BUILD_GENERAL1_SMALL
```
Comment (awsluja) [inline codebuild_specs/e2e_workflow_generated.yml:836]: if i remember correctly we were running into out-of-memory issues with this test, so if we see it failing we might also need to put this job on a medium VM ; small vm only has 3gb and is fine for most tests; 
https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-compute-types.html

**Review by awsluja — APPROVED** [review]

File: codebuild_specs/e2e_workflow_generated.yml (hunk 1 above)
Comment (sobolk) [reply codebuild_specs/e2e_workflow_generated.yml:836]: I kicked off a run. If that's the case I'll add option to customize compute per test.

Comment (sobolk) [reply codebuild_specs/e2e_workflow_generated.yml:836]: @awsluja looks like it runs on medium...

image

It seems that
```
      env:
        variables:
          compute-type: BUILD_GENERAL1_SMALL
```
is invalid yaml....
it should be 
```
      env:
        compute-type: BUILD_GENERAL1_SMALL
```

**Pull request merged**